### ISSUE TITLE
True Studio

### ISSUE THREAD

**Issue by gustavokcouto** [Lexatagan/LRNDIS#1]
Hello,

I'm trying to port the LRNDIS lybrary to attolic and stm32cube, I was working before on the code in the main repo, did this fork worked in the end?

**Comment by Lexatagan** [comment]
Hello, Gustavo. Yes, my fork worked in the end. What problem do you have?

**Comment by gustavokcouto** [comment]
windows 10 is reconizing it as a COM port instead of a network interface

**Comment by Lexatagan** [comment]
Hello, Gustavo. Yes, my fork worked in the end. What problem do you have?

**Comment by Lexatagan** [comment]
Yes, last week

**Comment by Lexatagan** [comment]
Yes, I tried STM32 device whith Windows 10 last week, and also got that problem. As far as I know this is common problem for many RNDIS devices. This is due to the new driver in Windows 10.  Problem solution - editing USB-descroptors according to rndiscmp.inf.
